File: 283. Move Zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        j=0
        for i in range(len(nums)):
            if nums[i]:
                if i!=j:
                    nums[j]=nums[i]
                    nums[i]=0
                j+=1
            
        

File: 283. Move Zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int j=0;
        int k=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]!=0){
                k=nums[j];
                nums[j]=nums[i];
                nums[i]=k;
                j++;
                
            }
        }
    }
}
